content/en/blog/how-to-use-fission-functions-with-postgresql-database/index.md

```diff
@@ -0,0 +1,357 @@
+By the end of this blog post, you would have learnt how to use PostgreSQL database with Fission functions.
+
+So let's get started!
+
+{{< imgproc votingapp-fission-function.png Fit "1000X1000" >}}
+Voting App using Fission functions with PostgreSQL Database
+{{< /imgproc >}}
+
+## Pre Requisites
+
+### Fission
+
+Before you start working on this demo, you need to ensure that you have Fission installed and running on your system.
+You can refer to our [Fission Installation](https://fission.io/docs/installation/) guide for more.
+
+### PostgreSQL
+
+You must also have a PostgreSQL instance in place. You can either run it on your system locally, in a docker container or in your Kubernetes cluster.
+
+Below are a few guides you can refer to for installing PostgreSQL:
+
+- Installing PostgreSQL on your local system - <a href="https://www.postgresql.org/download/" target="_blank">Postgresql.org</a>
+- PostgreSQL Docker container - <a href="https://hub.docker.com/_/postgres" target="_blank">Docker Hub</a>
+- PostgreSQL HELM charts - <a href="https://bitnami.com/stack/postgresql/helm" target="_blank">Bitnami</a> - *used in this example*
+
+<br>
+
+> *Note: If you have installed it using Helm Charts, please note down the service url and port number. This will be used to connect to the database from the function. You can get it by running  `helm status postgresql` in your terminal. You will get the following output.*
+
+```bash
+    NAME: postgresql
+    LAST DEPLOYED: Tue Dec 21 16:50:16 2021
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 1
+    TEST SUITE: None
+    NOTES:
+    CHART NAME: postgresql
+    CHART VERSION: 10.13.14
+    APP VERSION: 11.14.0
+
+    ** Please be patient while the chart is being deployed **
+
+    PostgreSQL can be accessed via port 5432 on the following DNS names from within your cluster:
+
+        postgresql.default.svc.cluster.local - Read/Write connection
+
+    To get the password for "postgres" run:
+
+        export POSTGRES_PASSWORD=$(kubectl get secret --namespace default postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+
+    To connect to your database run the following command:
+
+        kubectl run postgresql-client --rm --tty -i --restart='Never' --namespace default --image docker.io/bitnami/postgresql:11.14.0-debian-10-r17 --env="PGPASSWORD=$POSTGRES_PASSWORD" --command -- psql --host postgresql -U postgres -d postgres -p 5432
+
+    To connect to your database from outside the cluster execute the following commands:
+
+        kubectl port-forward --namespace default svc/postgresql 5432:5432 &
+        PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d postgres -p 5432
+
+```
+
+#### Database and Table Setup
+
+For this application, we have created a database named `votedb` and a table named `votebank`. You can use the following `sql` query to create the table:
+
+``` sql
```

Comment:
We need to indent line 90 and line 96 also along with the snippet. That way it get rendered properly in the page.
Same for above-embedded code snippets.

Reply:
fixed